src/cudafeatbin/compute-fbank-online-batched-cuda.cc

```diff
@@ -146,7 +146,7 @@ int main(int argc, char *argv[]) {
     int32_t ldf = max_chunk_frames;
 
     CudaOnlineBatchedSpectralFeatures fbank(feature_opts, max_chunk_frames,
-                                            num_channels, num_lanes);
+                                           num_channels, num_lanes);
```

Comment:
Oops. ×2 (the 2nd @ L256)
```suggestion
                                            num_channels, num_lanes);
```